apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/out.template.component.ts

```
@@ -94,14 +97,14 @@ export class OutTemplateComponent implements OnInit, OnDestroy, OnChanges {
             if (template && template.logoUniqueName) {
                 this.showLogo = true;
                 if (!this._invoiceUiDataService.isLogoUpdateInProgress) {
-                    this.logoSrc = ApiUrl + 'company/' + this.companyUniqueName + '/image/' + template.logoUniqueName;
+                    this.logoSrc = (config.ApiUrl || ApiUrl) + 'company/' + this.companyUniqueName + '/image/' + template.logoUniqueName;
                 }
             }
             if (template && template.sections) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)